data/moves/animations.asm

```diff
@@ -2082,6 +2082,7 @@ BattleAnim_Headbutt:
 	anim_ret
 
 BattleAnim_Tackle:
+; BUG: BattleAnim_Tackle reaches horizontal sprite limit (see docs/bugs_and_glitches.md)
```

Comment:
The function name and hardware internals aren't visible; what is the visible consequence that leads to this being a bug? "Tackle is missing part of its hit animation", maybe?

Reply:
Hmm... I can't think of anything better than that. That's pretty much what is occuring.  
  
\`\`\`  
; BUG: Tackle is missing part of its hit animation (see docs/bugs\_and\_glitches.md)  
\`\`\`